### PR TITLE
[Doc] Fix add Roboto font integration step to vite tutorial

### PR DESCRIPTION
**Pull request by WiXSL**
Related to #8407

### CODE REVIEW

**Comment by smeng9** [comment]
I think we also need to update https://github.com/marmelab/react-admin/blob/master/docs/Vite.md for changes in both https://github.com/marmelab/react-admin/pull/8415 and https://github.com/marmelab/react-admin/pull/8419

File: docs/Tutorial.md [hunk 1]
```diff
@@ -89,6 +89,28 @@ body {
 }
 ```
 
+In order to have the project using `Roboto` font, you have to add it to your index.html file
```
Comment (slax57) [inline docs/Tutorial.md:92]:
```suggestion
Lastly, add the `Roboto` font to the `index.html` file:
```

**Review by slax57 — CHANGES_REQUESTED** [review]
(very) minor change suggestion

File: docs/Tutorial.md [hunk 2]
```diff
@@ -89,6 +89,28 @@ body {
 }
 ```
 
+Lastly, add the `Roboto` font to the `index.html` file:
```
Comment (fzaninotto) [inline docs/Tutorial.md:92]: I'd put the custom css and the font after the initial screenshot, as it's strictly speaking not necessary to get the welcome screen (and it is more readable like that)

**Review by fzaninotto — CHANGES_REQUESTED** [review]
Don't forget to update the code in the examples/tutorial directory, too!

**Review by slax57 — APPROVED** [review]

**Review by fzaninotto — APPROVED** [review]

**Pull request merged**